resources/sdk/clisdkclient/templates/api.mustache

```diff
@@ -38,7 +39,7 @@ func Cmd{{classVarName}}() *cobra.Command { {{#operation}}
 
 {{#operation}}
 var {{operationId}}Cmd = &cobra.Command{
-	Use:   "{{operationId}}{{#pathParams}} [{{paramName}}{{#hasMore}}{{/hasMore}}]{{/pathParams}}",
+	Use:   "{{operationId}}{{#pathParams}} [{{paramName}}{{#hasMore}}{{/hasMore}}]{{/pathParams}}{{#queryParams}}{{#required}} [{{paramName}}]{{/required}}{{/queryParams}}",
```

Comment:
Probably best to leave this change out. This would confuse people into thinking it was a path param.
Marking the flag as required would be enough to let people know to add it

Reply:
Yeah I also thought about that and not entirely sure if we should have the required query parameter there. But for the usage, we use curly brackets for path params and square brackets for query params. And since it's telling users how to use the command, maybe we should include the required parameters, but using different syntax to represent query params? I'm okay with either one. Any thoughts on this? @carnellj

Reply:
Agree with Ronan.  This is an edge case.  Very few of the commands have required queryParameters.  This is probably a situation where someone forgot to add it to the body and quick added it as a queryParam

Reply:
Sure, I'll remove that query parameter part then.

Reply:
Fixed. Removed the query param section in usage.